packages/dappkit/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "baseUrl": ".",
     "declaration": true,
     "jsx": "preserve",
-    "lib": ["dom", "es2015", "es2016", "es2017", "es2018", "esnext"],
+    "lib": ["dom", "es2015", "es2016"],
```

Comment:
It would be nice to start inheriting typescript configs. Check contractkit one:

```
{
  "extends": "../typescript/tsconfig.library.json",
  "compilerOptions": {
    "rootDir": "src",
    "outDir": "lib"
  },
  "include": ["src", "types/"]
}
```